### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by cesarvargas00**
I've changed apple to orange.
I've done this to teach someone how to do a pull request. And also, because Apple is a computer brand name; and I like oranges better 👯‍♂️

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -159,7 +159,7 @@ end
 
 Grocery.new()
 #=> %Grocery{
-#     item: {"Apple", 4, false}
+#     item: {"Orange", 4, false}
```
Comment (expede) [inline README.md]: This won't actually produce this output. You need to update the definition's overwrite as well.

**Comment by expede** [comment]
> I've done this to teach someone how to do a pull request

Awesome; mentoring is always a good thing! Sadly this change currently breaks the example (I've pointed out where)

File: README.md (hunk 1 above)
Comment (cesarvargas00) [reply README.md]: Ok, I will overwrite that as well! Thank you for the patience 🙇

**Comment by cesarvargas00** [comment]
> Looks great! You get the...
> 
> 
> 
> Feel free to merge 👍
> 
> 
> 

I think I need write access to merge pull requests 🤔

**Comment by expede** [comment]
@cesarvargas00 It was already merged 👌

**Pull request merged**